avocado/utils/nvme.py

```
@@ -529,3 +530,121 @@
+
+
+def is_lockdown_supported(namespace):
+    """
+    Feteches information based on namespace
+
+    :rtype: boolean
+    """
+    lockdown_attr = get_nvme_sed_discover_parameters(namespace)
+    if lockdown_attr["Locking Supported"] == "Yes":
+        return True
+    return False
```

Ruff (0.12.2)

Return the condition `lockdown_attr["Locking Supported"] == "Yes"` directly

Replace with `return lockdown_attr["Locking Supported"] == "Yes"`

(SIM103)